src/checkstyle/checks/design/InterfaceCheck.hx

```diff
@@ -4,15 +4,17 @@ import checkstyle.token.TokenTree;
 import haxe.macro.Expr;
 import checkstyle.LintMessage.SeverityLevel;
 
-@name("InterfaceIsType")
-@desc("Checks for interfaces that does not contain any methods but only constants")
-class InterfaceIsTypeCheck extends Check {
+@name("Interface")
+@desc("Checks and enforces interface style (allow properties and methods or just methods")
+class InterfaceCheck extends Check {
 
 	public var allowMarkerInterfaces:Bool;
+	public var allowProperties:Bool;
```

Comment:
Finer-grained control than just "allow all properties" could be nice, which would be possible with the system I have in mind for https://github.com/adireddy/haxe-checkstyle/issues/54.